use @celo/celo-devchain for sdk tests needing a devchain

aaronmgdr opened the pull request:
### Description

Previously to run tests in sdk such as cli and contractkit that needed a mock chain to test against it was neccessary to build everything by hand each time. and required running ascript in protocol. this will not be possible once ck is moved out. and for now just speeds up everything. 



### Other improvements

Note before this i have not been able to run ck locally for a little while and now i can so major win.


### Tested


### Related issues

This is part of the move to untangle contractkit from protocol so that sdks can be move to a new repo. the other remaining are 

* replacing script that builds contractkits generated folder which contains Typescript Contracts with importing those same files from @celo/abis 
* remove function used in contractkit tests that is imported from protocol but also duplicated in contractkit itself 
* remove MetaTransactionWallet test which rely on files from protocol
* modify SortedOracles test which have same issue

### Backwards compatibility

n/a

### Documentation
n/a

aaronmgdr committed:
use @celo/celo-devchain package to as initial chain state rather than building it each time.

shazarre commented on packages/sdk/contractkit/src/test-utils/setup.global.ts:
```diff
@@ -8,8 +8,9 @@ import * as path from 'path'
 // If there is not, then your editor probably deleted it automatically.
 
 export default async function globalSetup() {
-  console.log('\nstarting ganache...')
```
you don't want to keep the log here?

shazarre reviewed (approved):
left one minor comment, otherwise looks good to me

aaronmgdr committed:
use new devchain

soloseng committed:
testing w/ contracts artifacts in devchain package

soloseng committed:
Bump version fix

soloseng committed:
Merge commit 'ff3b4b3a6b282600fbef48cf7c5908eb9c323efb' into aaronmgdr/divorce-protocol-ck-4

soloseng commented:
Not sure if intended, but `transaction-uri` is not being tested in CI

soloseng committed:
bump celo-devchain version

soloseng committed:
BUMP celo-devchain to stable version

aaronmgdr committed:
add back log feedback
more accurate const names
fix versions

aaronmgdr committed:
merge

aaronmgdr commented:
> Not sure if intended, but `transaction-uri` is not being tested in CI

ACK!

aaronmgdr committed:
LockedGold.getTotalPendingWithdrawalsCount requires contracts to be deployed at a higher version than us currently latest on celo mainnet.

so note this to help developers understand why instead of just a VM revert message

aaronmgdr committed:
dont ignore transaction-uri tests, these only take 3 seconds so its not worth separating them out.

aaronmgdr committed:
Merge branch 'master' into aaronmgdr/divorce-protocol-ck-4

soloseng committed:
bump celo/devchain version

aaronmgdr committed:
latest beter all around
